internal/rbd/rbd_util.go

```diff
@@ -1441,6 +1441,43 @@ func (rv *rbdVolume) isThickProvisioned() (bool, error) {
 	return thick, nil
 }
 
+// RepairThickProvision writes zero bytes to the volume so that it will be
+// completely allocated. In case the volume is already marked as
+// thick-provisioned, nothing will be done.
+func (rv *rbdVolume) RepairThickProvision() error {
```

Comment:
If this rewrites all the contents (with zeros) in the existing volume maybe can add a note here to use this carefully?

Reply:
Is `RepairThickProvision writes zero bytes to the volume` not clear enough?

Reply:
I thought of having a warning for using this function or clone. As it overwrites the content. but if you think its fine. we can leave it